test/jest-puppeteer-vuetify/package.json

```diff
@@ -1,9 +1,16 @@
 {
   "name": "jest-puppeteer-vuetify",
   "version": "0.1.0",
-  "main": "src/index.js",
+  "description": "Test utilities to make locating Vuetify components easier",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc"
+  },
   "dependencies": {
-    "@jest/globals": "*"
+    "@jest/globals": "*",
+    "@types/puppeteer": "^3.0.0",
```

Comment:
Do you foresee consumers also being written in TS? If so, they'd need `@types/puppeteer`, so it should be in `dependencies`. If consumers will just import the transpiled JS version of this, then `@types/puppeteer` should be a `devDependency` (as [endorsed by `DefinitelyTyped`](https://github.com/DefinitelyTyped/DefinitelyTyped#npm))

Reply:
I think it be unnecessarily prescriptive to require TS, so I will move this to a devDependency. Generally I think TS is less useful in test code, as there are much fewer moving parts, and the code being written is generally very linear and easy to read.